compiler/src/dmd/cxxfrontend.d

```diff
@@ -178,6 +179,8 @@ Dsymbols* include(Dsymbol d, Scope* sc)
     return dmd.dsymbolsem.include(d, sc);
 }
 
+bool isFuncHidden(ClassDeclaration* cd, FuncDeclaration* fd);
```

Comment:
this should be `isFuncHidden(ClassDeclaration cd, FuncDeclaration fd)` (note no `*`) as this is a D file.